conanfile.py

```diff
@@ -320,6 +324,32 @@ def build_configure(self):
             if self.settings.os == "Windows":
                 args.append('--enable-libmfx' if self.options.qsv else '--disable-libmfx')
 
+            if tools.cross_building(self.settings):
+                args.append('--enable-cross-compile')
+                if self.settings.os == "Android":
+                    args.append('--target-os=android')
+                    if "android_ndk_installer" in self.deps_env_info:
+                        ndk_env_info = self.deps_env_info["android_ndk_installer"]
+                        cc = ndk_env_info.CC
+                        cxx = ndk_env_info.CXX
+                        ndk_root = ndk_env_info.NDK_ROOT
+                    else:
+                        env = os.environ
+                        cc = env["CC"]
```

Comment:
I think we should use it unconditionally (not just for cross-building and not just for Android)
something like that (at top level):
```
if "CC" in os.environ:
    args.append('-cc=%s' % cc)
# same for CXX
```